xmlua/libxml2/c14n.lua

```diff
@@ -0,0 +1,46 @@
+local ffi = require("ffi")
+
+ffi.cdef[[
+  typedef enum {
+    XML_C14N_1_0 = 0, /* Original C14N 1.0 spec */
+    XML_C14N_EXCLUSIVE_1_0 = 1, /* Exclusive C14N 1.0 spec */
+    XML_C14N_1_1 = 2, /*  C14N 1.1 spec */
+  } xmlC14NMode;
```

Comment:
Could you copy the original `c14n.h` contents without indenting and formatting?
It's for easy to maintain.